app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from datetime import datetime
+
+start = datetime.now().second
+
+while True:
+    now = datetime.now()
+    while start == now.second:
+        now = datetime.now()
+
+    start = now.second
+
+    try:
+        file = open(f"app-{now.hour}_{now.minute}_{now.second}.log", "w")
+        file.write(str(now))
+    except IOError as e:
+        print(e)
```

Comment:
I don't understand for what you use exception handling here. If you create file in the directory.

Reply:
for example, when you create a file on the disk where we create this file, the free space suddenly runs out.
and this is just one of many examples